kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -37,9 +37,21 @@ def build(self):
         self.alpha = 5.
 
         # check the number of filters per octave
-        if self.Q < 1:
+        if np.any(np.array(self.Q) < 1):
             raise ValueError('Q should always be >= 1, got {}'.format(self.Q))
 
+        if isinstance(self.Q, int):
+            self.Q = (self.Q, 1)
+        elif isinstance(self.Q, tuple): 
+            if len(self.Q) == 1:
+                self.Q = self.Q + (1, )
+            elif len(self.Q) < 1 or len(self.Q) > 2: 
+                raise NotImplementedError("Q should be a 1-tuple or 2-tuple. "
```

Comment:
or an integer! we still allow `isinstance(self.Q, int)` (see above)

Reply:
done